tests/address_bar_and_search/test_tile_menu_options.py

```diff
@@ -93,29 +83,29 @@ def test_tile_context_menu_options(driver: Firefox, index: int, sponsored: bool)
     C1533798.2: Ensure that a website has the appropriate context menu actions in the tile.
     """
     # initialize objects
-    nav = Navigation(driver).open()
-    tabs = TabBar(driver)
+    newtab = AboutNewtab(driver).open()
+    sleep(3)  # allow page to load, waiting for image isn't enough
```

Comment:
I'm assuming it's not possible using the `wait_for_page_to_load`? : (

Reply:
yeah, I tried that. Sadly, the hard wait was more effective. We will eventually work on removing hard waits, but this was a bit of a disappointment.